build/README.md

```diff
@@ -1,60 +1,170 @@
+# Start with docker compose
+
+You can start ZITADEL with a simple docker compose up.
+
+The services are configured to restart if an error occurs. Until the essential services are started and executed some services panic, this is expected just give it some minutes to setup the database and execute migrations.
+
+```bash
+COMPOSE_DOCKER_CLI_BUILD=1 DOCKER_BUILDKIT=1 docker compose -f ./build/local/docker-compose-dev.yml --profile init --profile backend --profile frontend -p zitadel up
```

Comment:
when just starting this command on macOS 11.4 it will rise an error

![Screenshot 2021-05-26 at 21 30 55](https://user-images.githubusercontent.com/9879976/119720026-c623a380-be69-11eb-897c-2809912f3ba9.png)